src/ProductAttributePageTitle.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Allows optional customization of page title.
+ */
+class ProductAttributePageTitle {
+
+  public static function init() {
```

Comment:
This is introducing a dependency of ACF. It's installed on most websites nowadays, however I think this plugin might be installed on some sites without ACF.

Reply:
@wrux Shop standards is already using ACF.

Reply:
[Here](https://github.com/netzstrategen/wordpress-shop-standards/blob/master/acf/brands.hide-to-cart-button-product-categories-brands.json)